### PR TITLE
Obsolete foreman-compute

### PR DESCRIPTION
**Pull request by tbrisker**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -34,6 +34,8 @@ BuildArch:  noarch
 Conflicts: foreman-tasks < 0.11.0-2
 Conflicts: foreman-release-scl < 7-1
 
+Obsoletes: foreman-compute
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec]: Should this be
```suggestion
Obsoletes: foreman-compute < 1.22.0-0.9.develop
```

Comment (evgeni) [reply packages/foreman/foreman/foreman.spec]: or `< %{version}`

Comment (evgeni) [reply packages/foreman/foreman/foreman.spec]:
```
10:31 < ewoud> Zhenech: I think your suggestion is correct
10:33 < ewoud> either can work, but I think the latter one is a bit more robust
10:34 < ewoud> the explicit version number does show that at some point you could drop it more easily, but we don't really do that anyway
```

Comment (tbrisker) [reply packages/foreman/foreman/foreman.spec]: amended. in the future feel free to push to my branch, especially when i'm afk :)

**Pull request merged**